ros2_batch_job/__main__.py

```diff
@@ -333,6 +335,30 @@ def process_coverage(args, job):
     return 0
 
 
+def force_xunit2_in_pytest_ini_files():
+    for path in Path('.').rglob('pytest.ini'):
+        config = configparser.ConfigParser()
+        config.read(str(path))
+        if StrictVersion(pytest.__version__) < StrictVersion('6.0.0'):
```

Comment:
This condition should probably only be evaluated once rather than repeatedly during the loop.

Reply:
should be fixed by 057c1f7